src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs

```diff
@@ -999,5 +999,48 @@ public async Task<IActionResult> ReconfirmHasNotStopChangesAsync(ReconfirmHasNot
                 ApprenticeshipHashedId = viewModel.ApprenticeshipHashedId
             });
         }
+
+        [Route("{apprenticeshipHashedId}/details/confirmWhenApprenticeshipStopped")]
+        [HttpGet]
+        public async Task<IActionResult> ConfirmWhenApprenticeshipStopped(ConfirmWhenApprenticeshipStoppedRequest request)
+        {
+            var viewModel = await _modelMapper.Map<ConfirmWhenApprenticeshipStoppedViewModel>(request);
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        [Route("{apprenticeshipHashedId}/details/confirmWhenApprenticeshipStopped")]
+        public async Task<IActionResult> ConfirmWhenApprenticeshipStopped(ConfirmWhenApprenticeshipStoppedViewModel viewModel)
+        {
+            if (viewModel.IsCorrectStopDate.HasValue)

```

Comment:
this check is handled by the Model validation in `ConfirmWhenApprenticeshipStoppedViewModelValidator`

Reply:
changed